admin_guide/out_of_resource_handling.adoc

```diff
@@ -146,26 +148,9 @@ eviction signal.
 |Available memory on the node has satisfied an eviction threshold.
 |===
 
-The node continues to report node status updates at the frequency specified by
-the `node-status-update-frequency` argument, which defaults to *10s*.
-
-[[out-of-resource-oscillation-of-node-conditions]]
-=== Oscillation of Node Conditions
-
-If a node is oscillating above and below a soft eviction threshold, but not
```

Comment:
I would not nest this below Troubleshooting.  I would keep it after Eviction Thresholds in a dedicated section.